### PR TITLE
Fix documentation links

### PR DESCRIPTION
**Pull request by dopsi**

### CODE REVIEW

File: src/models/mod.rs [hunk 1]
```diff
@@ -1,5 +1,7 @@
 //! This module will contain different models that can be used to perform certain calculations. E.g.
-//! Gates, like [`AndGate`], [`Mux`], [`Switch`], but also complex gates, like ['Alu`] or [`Rom1kx8`].
+//! Gates, like [`AndGate`](crate::models::gates::AndGate), [`Mux`](crate::models::gates::Mux),
+//! [`Switch`](crate::models::inputs::Switch), but also complex gates, like [`Alu`] or
```
Comment (hellow554) [inline src/models/mod.rs]: `Alu` is missing as well

Comment (dopsi) [reply src/models/mod.rs]: It is not yet present in the code, should I remove it?

Comment (hellow554) [reply src/models/mod.rs]: Ups 😅 yes, please remove it.

**Comment by hellow554** [comment]
LGTM. Can you squash it and force push pleae?

**Comment by dopsi** [comment]
Done

**Comment by hellow554** [comment]
Thank you very much

**Pull request merged**